packages/paste-core/components/corner-ornament/src/CornerOrnamentElement.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { useUID } from "@twilio-paste/uid-library";
+import * as React from "react";
+
+import { useCornerOrnamentContext } from "./CornerOrnamentContext";
+import { BadgeBottomEndPath, DotBottomEndPath } from "./Masks";
+import { CornerOrnamentElementProps, CornerOrnamentPosition, CornerOrnamentType } from "./types";
+
+export const CornerOrnamentElement = React.forwardRef<HTMLDivElement, CornerOrnamentElementProps>(
```

Comment:
suggestion(non-blocking): what about calling this `CornerOrnamentBase`? I keep forgetting whether `CornerOrnamentElement` refers to the base element or the ornament element

Reply:
todo: Lets go with base